test/e2e_node/BUILD

```diff
@@ -135,6 +135,7 @@ go_test(
         "startup_probe_test.go",
         "summary_test.go",
         "system_node_critical_test.go",
+        "taints.go"
```

Comment:
I think you're missing a comma here :) which is I think responsible for pretty much all of the test failures :)

Reply:
yeah, not sure if this was done manually but if so running `./hack/update-bazel.sh` is how these should be updated

Reply:
Is BUILD file can be generated automatically? Any script can do it?

Reply:
Yup, `hack/update-bazel.sh` is the script which will do it.